filament/src/RenderPass.cpp

```diff
@@ -577,6 +580,16 @@ void RenderPass::Executor::recordDriverCommands(backend::DriverApi& driver,
                         skinning.offset * sizeof(PerRenderableUibBone),
                         CONFIG_MAX_BONE_COUNT * sizeof(PerRenderableUibBone));
             }
+
+            auto morphTargetBuffer = info.morphTargetBuffer;
+            if (UTILS_UNLIKELY(morphTargetBuffer)) {
+                auto morphing = soaMorphing[info.index];
+                assert_invariant(morphing.handle);
+                driver.bindUniformBuffer(BindingPoints::PER_RENDERABLE_MORPHING, morphing.handle);
```

Comment:
Add a comment that instead of using a UBO per primitive, we could also have a single UBO for all primitives and use `bindUniformBufferRange` which might be more efficient.

Reply:
It's done. It's a good point!